src/test/java/tests/SignUpTest.java

```diff
@@ -0,0 +1,50 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.BasePage;
+import pages.HomePage;
+import pages.SignupPage;
+
+public class SignUpTest extends BaseTest {
+    SignupPage page;
+    HomePage home;
+    BasePage base;
+
+    @BeforeMethod
+    public void signUpPage(){
+        base = new BasePage(driver);
+        home = new HomePage(driver);
+        page = new SignupPage(driver);
+        home.openHomePage()
+                .toSignUpPage();
+        String expected = "Kwidos";
+        String actual = driver.getTitle();
+        Assert.assertEquals(actual, expected);
+    }
+
+    @Test
+    public void emptySignUp(){
+        page.emptySignup();
+        System.out.println("Empty Signup Success");
+        //the Create Account Button is disabled if
+        //the credential is incorrect
```

Comment:
What does verify your test?

Reply:
@mobilman89
It's impossible to write an assertion if the signup page does not let you do a negative test case.